point_of_sale/restaurant/restaurant.rst

```diff
@@ -0,0 +1,118 @@
+==================
+Manage your tables
+==================
+
+Restaurants have very specific needs that shops don’t have. That’s why
+Odoo gives restaurant owners several unique features to help them manage
+their business in the best possible way.
```

Comment:
```suggestion
Restaurants have specific needs that shops don’t have. That’s why
Odoo gives restaurant owners several unique features to help them manage
their business in the best possible way.
```

This kind of suggestion comes from me reading the book Style, and he says we should not modify absolute words, unique, final or complete with very, more, quite and so on (:

Reply:
Never had the chance to read Style, so I didn't know ^^
Thanks!